test/fixtures/all.env

```diff
@@ -0,0 +1,57 @@
+#
+# env file for testing that contains all possible and impossible values
+#
+
+# global
+HLX_LOG_LEVEL = debug
+HLX_LOG_FILE = test.log
+
+# deploy + package + clean + build + up
+HLX_TARGET = foo
+
+# build + up
+HLX_CACHE = true
+HLX_MINIFY = true
+HLX_FILES = *.htl,*.js
```

Comment:
What happen if I change this config ? can I add more files ? Is this tested somewhere ?

Reply:
> What happen if I change this config ? can I add more files ? Is this tested somewhere ?

well, it's the same as specifying the files in `hlx up` or `hlx build`. eg:

```
hlx up ./src/foo/*.js
```

it is implicitly tested via:
https://github.com/adobe/helix-cli/blob/v0.13.14/test/testBuildCmd.js#L41
and
https://github.com/adobe/helix-cli/blob/v0.13.14/test/testBuildCli.js#L93